club/teams/tenure 24-25.md

```diff
@@ -23,9 +23,35 @@ nav_order: 5
 | ---- | ---------- | ----- | ----- |
 |      |            |       |       |
 
+
 ### Secretaries
 
-| Name | Department | Batch | Email |
-| ---- | ---------- | ----- | ----- |
-|      |            |       |       |
+| Name                | Department                       | Batch|   Email                                                  |
```

Comment:
@tanish-singla please verify if the details are correct and complete. Rest seems fine to me